Source/JavaScriptCore/inspector/remote/RemoteInspectionTarget.h

```diff
@@ -31,15 +31,19 @@
 #include "RemoteControllableTarget.h"
 #include <wtf/ProcessID.h>
 #include <wtf/RetainPtr.h>
+#include <wtf/ThreadSafeRefCounted.h>
 #include <wtf/TypeCasts.h>
 #include <wtf/text/WTFString.h>
 
 namespace Inspector {
 
 class FrontendChannel;
 
-class JS_EXPORT_PRIVATE RemoteInspectionTarget : public RemoteControllableTarget {
+class JS_EXPORT_PRIVATE RemoteInspectionTarget : public RemoteControllableTarget, public ThreadSafeRefCounted<RemoteInspectionTarget> {
 public:
+    void ref() const { ThreadSafeRefCounted<RemoteInspectionTarget>::ref(); }
```

Comment:
I think this function should be marked as `final` given that I believe it is a virtual function override.